### ISSUE TITLE
Dynamic iFrame content

### ISSUE THREAD

**Issue by K-Thompson** [noelboss/featherlight#256]
What kind of issue is this? (put 'x' between the square brackets)

 - [ ] Question. This issue tracker is not the place for questions. If you want to ask how to do
       something, or to understand why something isn't working the way you expect it to, use
       http://stackoverflow.com/questions/ask .
       Provide working code, starting from http://jsfiddle.net/JNsu6/15/.
       We monitor the tag `featherlight.js`.

 - [ X] Bug report. If you’ve found a bug, you must provide a minimal example in a CodePen,
       starting from http://jsfiddle.net/JNsu6/15/ .

 - [X ] Feature Request. Make sure there's no good way to do what you want first;
       consider asking on http://stackoverflow.com/questions/ask first.

Thank you for Featherlight. You might be interested to know that I'm want to replace Fancybox with Featherlight due to Fancybox's license (can't be used in Wordpress plugins).

My use case is the display of dynamic content in an `iframe`.

Though the URL is valid, this fails:

      $.featherlight({iframe: 'about:blank', iframeWidth: '96%' });

Here's my my solution including a slight modification to Featherlight:

[dynamic content in an iframe](http://stackoverflow.com/questions/40335303/featherlight-javascript-source-for-dynamic-content/40358913#40358913)

The solution calls for adding an `id` attribute to the `iframe`. However, I'm not sure if this might break other possible uses, so I'm not making a pull request. Please consider this use case and adding support for it.

**Comment by marcandre** [comment]
Thanks for reporting this, and providing a working example.

Sorry about the bug for `about:blank`. Fixed and released.

I'm not against setting an ID per say, but I don't see why. You could use `$('.featherlight iframe')` to locate it, no?

**Comment by K-Thompson** [comment]
Wow, thanks for the fix.

> $('.featherlight iframe')

Silly me. I'm not in the habit of thinking in jQuery terms. I'll give this a go.

**Comment by K-Thompson** [comment]
@marcandre 

Not being particularly familiar with jQuery, I had to do some research to figure out how to write content to the iframe element returned by the JQuery class selector. Using this [accepted SO answer](http://stackoverflow.com/questions/21795761/insert-content-into-iframe), I came up with this:

```
		var s = '<!DOCTYPE html><html><head><meta charset="UTF-8"><title>Title of the document</title></head><body><p>Content of the document......</p></body></html>';
		$.featherlight({iframe: 'about:blank'});
		var $iframe = $('.featherlight iframe');
		$iframe.ready(function () {
			$iframe.contents().find("body").append(s);
		});
```

(The first sentence in the SO answer is particularly interesting ;-)

For my case, I'm going to leave the modification that inserts an `id`. As I mention, I'm porting code and the pages displayed within the `iframe` have buttons on them. To initialize event listeners, the buttons are found via the iframe's id. This code is already well tested.
